altair/vegalite/v5/api.py

```diff
@@ -603,9 +600,6 @@ def selection_point(
     encodings : List[str] (optional)
         A list of encoding channels. The corresponding data field values
         must match for a data tuple to fall within the selection.
-    fields : List[str] (optional)
-        A list of field names whose values must match for a data tuple to
-        fall within the selection.
```

Comment:
Hmm I actually think we should add `fields` as a named parameter in the function signature instead since it can indeed be used for point selections. @ChristopherDavisUCI Does this makes sense to you to or do you remember if there was any particular reason it was left out in https://github.com/altair-viz/altair/pull/2908?

Reply:
Cool, makes sense: 2b94e606